youtube_dl_server/server.py

```diff
@@ -29,6 +29,14 @@ def main():
         help='The host the server will use. The default is: %(default)s',
     )
 
+    parser.add_argument(
+        '--number-processes',
+        default=5,
```

Comment:
Ummm, although the default value using gunicorn is `1` recommended use `2|4 x $(NUM_CORES)`. This formula mean that the minimum recommended value is `2`...

I think the best default is `2` for this project:
- Avoids the problem of blocking the server with requests that need wait (gunicorn will not have this problem)
- Don't consume too many resources, after all is the minimum recommended.

Anyway, it would be nice add to the help of `--number-processes` that should take into account the number of cores